### ISSUE TITLE
Update PropTypes for Metadata Container and Metadata Component

### ISSUE THREAD

**Issue by j-pendlebury** [bbc/simorgh#2020]
~Blocked on #1906~

**Is your feature request related to a problem? Please describe.**
Our work on refactoring Metadata Container change the values for some of the props passed and made others only relevant for articles or frontpages. We need to update propTypes used in 
* Metadata Container
* Metadata Component
* LinkedData Component

to reflect the new props we receive. 

**Describe the solution you'd like**
1) Use different propType files for Metadata Container (e.g. we have [metadata propTypes](https://github.com/bbc/simorgh/blob/metadata-refactor/src/app/models/propTypes/metadata/index.js#L3), we could have articleMetadata and frontpageMetadata, etc. Can use prop-types' `oneOf` to assert that we have either.

2) Fix the propTypes by requiring different props depending on asset type ('article' or 'IDX'), e.g. require optimoUrn for article, not needed for frontpage.

**Describe alternatives you've considered**
Leave propTypes as they are if we can't find a solution, but we probably shouldn't do that.

**Testing notes**
Check that you don't get propType warnings.

Dev insight: -

**Additional context**
